configure.ac

```diff
@@ -242,7 +241,8 @@ AS_CASE(["x$with_fuzzing"],
         [AC_MSG_ERROR([Bad value for --with-fuzzing])])
 AM_CONDITIONAL([ENABLE_FUZZING],[test "x$with_fuzzing" != "xnone"])
 AS_IF([test "x$with_fuzzing" != "xnone"],
-      [AS_IF([test "x$enable_tcti_fuzzing" = xno],
+      [AC_PROG_CXX
```

Comment:
This throws warnings for me:
configure.ac:244: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required

Reply:
and shouldn't there be a comma after AC_PROG_CXX?

Reply:
> This throws warnings for me:
> configure.ac:244: warning: AC_REQUIRE: `AC_PROG_CXX' was expanded before it was required

I'm seeing this too, I just ran in CI I didn't run locally so I missed this. Sorry.